tests/baselines/reference/nodeModulesTripleSlashReferenceModeDeclarationEmit3(module=node16).js

```diff
@@ -25,15 +25,54 @@ declare global {
     "type": "module"
 }
 //// [index.ts]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" resolution-mode="require" preserve="true" />
 export interface LocalInterface extends RequireInterface {}
 
 //// [index.js]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" resolution-mode="require" preserve="true" />
 export {};
 
 
 //// [index.d.ts]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" preserve="true" />

```

Comment:
Dont you need resolution-mode in d.ts and i think thats why its causing it not to include ?

Reply:
Woah, weird. Thanks for noticing. It's written in the original code so there's something else wrong in what I changed. 

![image](https://github.com/microsoft/TypeScript/assets/5341706/241f3bb0-f80f-4951-9eb6-5a41bfc1ee87)